packages/docusaurus-utils/package.json

```diff
@@ -51,6 +51,6 @@
     "@babel/core": "^7.0.0",
     "react": "*",
     "react-dom": "*",
-    "webpack": "5.x"
+    "webpack": "^5.0.0"
```

Comment:
`5.x` is the same as `^5.0.0` as I understand?

Reply:
I wasn't sure. Was thinking better to standardize the version style. That was the only one using an `x`

Reply:
Ah, sure